src/mame/drivers/rainbow.cpp

```diff
@@ -2868,7 +2863,7 @@ WRITE8_MEMBER(rainbow_state::GDC_EXTRA_REGISTER_w)
 
 				OPTION_RESET_PATTERNS
 			}
-			m_PORT50  = data;
+			//m_PORT50  = data;
```

Comment:
Commented-out code needs an accompanying explanatory comment, otherwise no-one will know why it's commented out and not removed.